golang/cosmos/x/vbank/client/rest/tx.go

```diff
@@ -0,0 +1,222 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/cosmos/cosmos-sdk/x/distribution/client/common"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+)
+
+type (
+	withdrawRewardsReq struct {
+		BaseReq rest.BaseReq `json:"base_req" yaml:"base_req"`
+	}
+
+	setWithdrawalAddrReq struct {
+		BaseReq         rest.BaseReq   `json:"base_req" yaml:"base_req"`
+		WithdrawAddress sdk.AccAddress `json:"withdraw_address" yaml:"withdraw_address"`
+	}
+
+	fundCommunityPoolReq struct {
+		BaseReq rest.BaseReq `json:"base_req" yaml:"base_req"`
+		Amount  sdk.Coins    `json:"amount" yaml:"amount"`
+	}
+)
+
+func registerTxHandlers(clientCtx client.Context, r *mux.Router) {
```

Comment:
Here also, if the squatting on `/distribution/...` is intentional, document it.

Reply:
Deleted.